### PR TITLE
OCM-6166 | feat: add create_admin_user in rosa module

### PR DESCRIPTION
**Pull request by marcolan018**
https://issues.redhat.com/browse/OCM-6166
Signed-off-by: marcolan018 <llan@redhat.com>

### CODE REVIEW

**Comment by nirarg** [comment]
/hold 
This is v1.5.1 changes
Need to wait to all v1.5.0 documentation changes to be merged first
The Jira ticket for the documentation changes: [OCM-3300](https://issues.redhat.com/browse/OCM-3300)

File: modules/rosa-cluster-classic/main.tf [hunk 1]
```diff
@@ -28,6 +28,7 @@ locals {
     oidc_config_id       = var.oidc_config_id
   }
   aws_account_arn = var.aws_account_arn == null ? data.aws_caller_identity.current[0].arn : var.aws_account_arn
+  create_admin_user = var.create_admin_user
```
Comment (nirarg) [inline modules/rosa-cluster-classic/main.tf:31]: This local is never used

**Comment by nirarg** [comment]
@marcolan018 please rebase and than we will continue with merging
/hold cancel

**Comment by marcolan018** [comment]
@nirarg completed the rebase

File: modules/rosa-cluster-classic/main.tf [hunk 2]
```diff
@@ -53,6 +54,7 @@ resource "rhcs_cluster_rosa_classic" "rosa_classic_cluster" {
     )
   )
   multi_az             = var.multi_az
+  create_admin_user    = var.create_admin_user
```
[the remark this answers is not in the document]
Comment (marcolan018) [reply modules/rosa-cluster-classic/main.tf]: changed to local.create_admin_user, thanks for the comment. Please re-review the commit

**Pull request merged**